projects/helper/tokenMapping.js

```diff
@@ -83,7 +83,7 @@ const fixBalancesTokens = {
   },
   flow: {
     '0xd3bF53DAC106A0290B0483EcBC89d40FcC961f3e': { coingeckoId: 'flow', decimals: 18 },
-    '0x1b97100ea1d7126c4d60027e231ea4cb25314bdb': { coingeckoId: 'ankr-staked-flow', decimals: 18 },
+    '0x1b97100eA1D7126C4d60027e231EA4CB25314bdb': { coingeckoId: 'ankr-staked-flow', decimals: 18 },
```

Comment:
case shouldnt matter

Reply:
From my point of view, it made a difference because if we use Morho-blue adapter creation approach, after getting all the tokens locked in the smart contract, they have the format “flow:0x1b97100eA1D7126C4d60027e231EA4CB25314bdb” and we had to manually change it to “coingecko:ankr-staked-flow” but since they have different cases, it was difficult to compare and find the correct coingeckoId for a particular token address.

Reply:
weird the code autoconverts all evm addresses to lowercase